pkgs/development/python-modules/krb5/default.nix

```
@@ -0,0 +1,41 @@
+{
+  lib,
+  pkgs,
+  buildPythonPackage,
+  cython,
+  fetchPypi,
+  k5test,
+  pytestCheckHook,
+  pythonOlder,
+}:
+
+buildPythonPackage rec {
+  pname = "krb5";
+  version = "0.6.0";
```

```suggestion
  version = "0.6.0";
  pyproject = true;
```
and add `build-system`

---

Done